applications/GeoMechanicsApplication/tests/cpp_tests/test_generalized_newmark_scheme.cpp

```diff
@@ -11,24 +11,38 @@
 //
 
 #include "custom_strategies/schemes/generalized_newmark_scheme.hpp"
-#include "geo_mechanics_application_variables.h"
 #include "spaces/ublas_space.h"
 #include "testing/testing.h"
 
 using namespace Kratos;
 using SparseSpaceType = UblasSpace<double, CompressedMatrix, Vector>;
-using LocalSpaceType = UblasSpace<double, Matrix, Vector>;
+using LocalSpaceType  = UblasSpace<double, Matrix, Vector>;
 
-namespace Kratos::Testing {
+namespace Kratos::Testing
+{
+
+// We need this class to test all non-abstract functions of the
+// GeneralizedNewmarkScheme class. We cannot use the
+// GeneralizedNewmarkScheme class directly, because it is abstract.
+class ConcreteGeneralizedNewmarkScheme : public GeneralizedNewmarkScheme<SparseSpaceType, LocalSpaceType>
+{
+public:
+    using GeneralizedNewmarkScheme::GeneralizedNewmarkScheme;
+
+protected:
+    void UpdateVariablesDerivatives(ModelPart& rModelPart) override
+    {
+        // Intentionally left empty
+    }
+};
 
 KRATOS_TEST_CASE_IN_SUITE(ForInvalidTheta_CheckNewmarkScheme_Throws, KratosGeoMechanicsFastSuite)
 {
     constexpr double invalid_theta = -2.0;
-    using SchemeType = GeneralizedNewmarkScheme<SparseSpaceType, LocalSpaceType>;
+    using SchemeType               = ConcreteGeneralizedNewmarkScheme;
```

Comment:
Since there are no template parameters needed, I think we could remove the using SchemeType and just use `ConcreteGeneralizedNewmarkScheme` directly in the `KRATOS_EXPECT_EXCEPTION_IS_THROWN`

Reply:
done